File: cybrina.py
# Script generate an error on OS X about timeout
# same lines work great on OS X python command console
import serial
#import time

ser = serial.Serial("/dev/cu.usbserial", rate=115200, timeout=1)
print ser.portstr
"""
f_string = "choregraphie.txt"
f_open = open(f_string,'r')
for line in f_open.readlines():
	ser.write(line+"\n\r")
	print line
	time.sleep(200)
	print "fini"
"""
ser.close()
# ser.write("#0 P1500 T500\n")

File: cybrina.rb
#
# cybrina.rb to access serial port on MacOS
#
# needs serialport for ruby in **ruby1.8**
# To install on OS X follow http://ruby-serialport.rubyforge.org/
# on bash : sudo gem install ruby-serialport
# Launch with : ruby cybrina.rb
require 'rubygems'
require 'serialport'
sp = SerialPort.new "/dev/tty.usbserial", 115200
File.open "choregraphie.txt" do |file|
 file.each_line do |line|
 	sp.write(line + "\r\n")  
 	sleep 0.2
 end
end